packages/authentication/spfx/package-lock.json

```diff
@@ -0,0 +1,462 @@
+{
+    "name": "@microsoft/kiota-authentication-spfx",
```

Comment:
we're using lerna here, which means the libs don't have a package lock (managed by lerna).
remove this file.
add any additional package this lib needs to the main package.json with yarn